src/components/ListItem/ListItemWithImage.js

```diff
@@ -93,6 +93,7 @@ type Props = {
   badge?: string,
   iconBackgroundColor?: string,
   iconBorder?: boolean,
+  address?: string,
```

Comment:
Can't find any usage of that variable in that file. Why do you need to pass it and where do you use it?

Reply:
shouldComponentUpdate had logic comparing the props and failing to recognize dofferent accounts

Reply:
you're right here.. I'm just afraid someone will remove that property as unused in the future and we will come back to the previous issue